### DISCUSSION TITLE
Not able to lazy load remote in a Nx monorepo with module federation

### DISCUSSION THREAD

**Discussion by stefanellid-mms** [webpack/webpack#16885] [Q&A]
Hi!

I'm trying to implement the dynamic import using this kind of suggestion (https://webpack.js.org/concepts/module-federation/#dynamic-remote-containers) but even when the remoteEntry.js is loaded, the application breaks trying to find the remote scope in the window object.

I used the `@module-federation/utilities` package which comes with the dynamic load implementation.

The console prompts:

`Uncaught Error: Remote loaded successfully but <module-name> could not be found! Verify that the name is correct in the Webpack configuration!`

**Comment by alexander-akait** [comment]
Please ask this question in `@module-federation/utilities` repo, thank you

Also never remove the issue template, otherwise noone can help you
